addons/mail/controllers/webclient.py

```diff
@@ -89,5 +89,5 @@ def _process_request_for_internal_user(self, store, **kwargs):
             })
         if kwargs.get("canned_responses"):
             field_names = ["source", "substitution"]
-            canned_responses = request.env["mail.shortcode"].sudo().search_read([], field_names)
+            canned_responses = request.env["mail.canned.response"].search_read([], field_names)
```

Comment:
This is for admin returning the canned response of everyone (even non shared).
I'll update to apply the same kind of filter as the default one in the view.